packages/firestore/src/local/simple_db.ts

```diff
@@ -209,6 +209,7 @@ export class SimpleDb {
   static isMockPersistence(): boolean {
     return (
       typeof process !== 'undefined' &&
+      typeof process.env !== 'undefined' &&
       process.env.USE_MOCK_PERSISTENCE === 'YES'
```

Comment:
Does `process?.env?. USE_MOCK_PERSISTENCE === YES` work (you might want to look at the generated code)?

Reply:
ah yes, the newest TS tech!